test/controllers/feed_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "test_helper"
+
+class TalksControllerTest < ActionDispatch::IntegrationTest
```

Comment:
Regarding the test suite, I made sure that these new tests passes and all is well, though on my local machine all test suite doesn't work, There are quite a few errors which seems to be not related to my changes even if I take in account fixtures.

<img width="1003" alt="image" src="https://github.com/adrienpoly/rubyvideo/assets/11750685/f9de7489-315b-4184-b06d-b5d38e7f5249">

<img width="1253" alt="image" src="https://github.com/adrienpoly/rubyvideo/assets/11750685/b41b9298-a29e-437b-a416-02f29bdda14f">


Is this a known issue?